addon/webextension/background/main.js

```diff
@@ -82,6 +94,10 @@ window.main = (function () {
     }
   });
 
+  communication.register("closeSelector", sender => {
```

Comment:
I needed this to signal that the icon should go back to grey since there's more ways to get out of the selector ui than getting in. Originally this did more but I narrowed it down to this.